src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -537,6 +544,24 @@ public GetYourTranscriptsResponse getYourTranscripts(Integer userId) {
     }
 
     @Override
+    public TranscriptionTranscriberCountsResponse getTranscriptionTranscriberCounts(Integer userId) {
+
+        UserAccountEntity user = userAccountRepository.findByRoleAndUserId(TRANSCRIBER.getId(), userId);
+        if (user == null) {
+            throw new DartsApiException(USER_NOT_TRANSCRIBER);
+        }
+
+        List<Integer> courthouseIds = transcriberTranscriptsQuery.getAuthorisedCourthouses(userId, 4);
+
+        Integer numUnassigned = transcriberTranscriptsQuery.getTranscriptionsCountForCourthouses(courthouseIds, APPROVED.getId(), 0);
```

Comment:
The hardcoded `0` bit for userId doesn't look right to me, please check

Reply:
yes. not sure where that one came from should be userId